pkg/build/nodeimage/imageimporter.go

```diff
@@ -48,17 +50,25 @@ func (c *containerdImporter) End() error {
 
 func (c *containerdImporter) Pull(image, platform string) error {
 	return c.containerCmder.Command(
-		"ctr", "--namespace=k8s.io", "content", "fetch", "--platform="+platform, image,
+		"ctr", "--namespace="+commonNS, "content", "fetch", "--platform="+platform, image,
```

Comment:
`+commonNS` is harder to read / obscures the value and is unrelated to this PR, please revert this part